src/components/commons/Container.tsx

```diff
@@ -8,10 +8,12 @@ export function Container(
   props: PropsWithChildren<{ className?: string; bgClassName?: string }>
 ): JSX.Element {
   return (
-    <div className={classNames("w-full", props.bgClassName)}>
+    <div
+      className={classNames("xl:mx-auto xl:container flex", props.bgClassName)}
+    >
       <div
         className={classNames(
-          "mx-6 md:mx-12 xl:mx-[120px] 2xl:mx-[300px]",
+          "mx-6 md:mx-12 xl:mx-[120px] 2xl:mx-[300px] w-full",
```

Comment:
sorry curious qn are we able to do it without creating an additional outer div 

but just in this div to have the mx-auto or will it affect the dapp cards

Reply:
It works well as it is~